validator_key_cache/Cargo.toml

```diff
@@ -9,7 +9,8 @@ anyhow = { workspace = true }
 bls = { workspace = true }
 eip_2335 = { workspace = true }
 fs-err = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order